packages/components/organisms/f-content-cards/src/components/ContentCards.js

```diff
@@ -69,19 +94,56 @@ export default {
         const locale = globalisationServices.getLocale(tenantConfigs, this.locale, this.$i18n);
         const localeConfig = tenantConfigs[locale];
 
+        const options = {
+            root: null,
+            rootMargin: '0px',
+            threshold: 0.7
+        };
+
+        const observerCallback = entries => {
+            entries.forEach(entry => {
+                const { card } = entry.target.__vue__.$options.propsData;
+                const { $el } = entry.target.__vue__;
+
+                this.cardObserver = {
+                    ...this.cardObserver,
+                    [card.id]: {
+                        isIntersecting: entry.isIntersecting,
+                        ...(entry.isIntersecting ? { intersectionTime: new Date() } : { intersectionTime: null })
+                    }
+                };
+
+                if (entry.isIntersecting) {
+                    this.handleCardView({
+                        ...card,
+                        metadata: {
+                            coordinates: {
+                                x: this.xOffsets.findIndex(offset => offset === $el.offsetLeft) + 1,
```

Comment:
This returns the index of the coordinate + 1, rather than the actual coordinate, right?

Reply:
yes they wanted index + 1

Reply:
Ah right, I finally understood this, so this turns pixel offsets into grid coordinates, i.e., 1st column, 2nd row